README.md

```diff
@@ -193,6 +193,15 @@ For detailed instructions on how to debug Triton's frontend, please refer to thi
   separated values to be specified (eg
   `TRITON_LLVM_DEBUG_ONLY="tritongpu-remove-layout-conversions` or
   `TRITON_LLVM_DEBUG_ONLY="tritongpu-remove-layout-conversions,regalloc"`).
+- `TRITON_ENABLE_ASAN=1` invokes the LLVM address sanitizer for
+  memory leak and out of bounds access detection. Currently only supported on the AMD
+  backend. This must be run using the ASAN libraries documented [here](https://rocm.docs.amd.com/projects/llvm-project/en/latest/conceptual/using-gpu-sanitizer.html).
+
+  When enabling the address sanitizer it is recommended to disable various memory caching strategies
```

Comment:
Maybe we can simplify by pointing to the test: "When enabling the address sanitizer it is recommended to disable various memory caching strategies both within the ROCm stack and PyTorch. See [this test](python/test/unit/test_address_sanitizer.py) for it."

This way it's also always up to date.

Reply:
We'll have to do this after this is merged since it will immediately create a broken sym link and break the CI.